docs/server/commands.rst

```diff
@@ -166,12 +166,20 @@ clear_stats
 
 Clear stats cache data.
 
+.. warning:: You must first **stop the workers**.
+
 Make use of :djadmin:`clear_stats` in cases where you want to remove all stats
-data.  Such a case may be where you want to recalculate stats after a change
+data. Such a case may be where you want to recalculate stats after a change
 to checks or wordcount calculations.  While it should be fine to run
 :djadmin:`refresh_stats` or :djadmin:`calculate_checks`, by first running
 :djadmin:`clear_stats` you can be sure that the stats are calculated from
 scratch.
+Use the :option:`--clear-rq` option to clear all RQ data (hanging jobs and
+dirty flags).
```

Comment:
This is docs, and after reading I am still not sure what "all RQ data" means. Do we flush revision? Do we flush all data contained on any Redis database? Pending jobs? stats? Anything on the queues?